x/wasm/keeper/keeper_mpt.go

```diff
@@ -0,0 +1,61 @@
+package keeper
+
+import (
+	"fmt"
+	"github.com/ethereum/go-ethereum/common"
+	ethtypes "github.com/ethereum/go-ethereum/core/types"
+	apptypes "github.com/okx/okbchain/app/types"
+	clientcontext "github.com/okx/okbchain/libs/cosmos-sdk/client/context"
+	"github.com/okx/okbchain/libs/cosmos-sdk/store/mpt"
+	sdk "github.com/okx/okbchain/libs/cosmos-sdk/types"
+	"github.com/okx/okbchain/libs/cosmos-sdk/x/auth"
+	"github.com/okx/okbchain/x/wasm/watcher"
+	"log"
+)
+
+func (k Keeper) getStorageStore(ctx sdk.Context, acc sdk.WasmAddress) sdk.KVStore {
+	account := k.accountKeeper.GetAccount(ctx, sdk.WasmToAccAddress(acc))
```

Comment:
evm has a account cache, it is better to add an account cache to wasm module later in another pr